src/overlays/actors/ovl_En_Horse/z_en_horse.c

```diff
@@ -3207,7 +3207,7 @@ void EnHorse_UpdateBgCheckInfo(EnHorse* this, PlayState* play) {
         }
 
         dynaPoly = DynaPoly_GetActor(&play->colCtx, bgId);
-        if ((this->stateFlags & ENHORSE_FLAG_26) && ((dynaPoly && dynaPoly->actor.id != 0x108) || dynaPoly == 0)) {
+        if ((this->stateFlags & ENHORSE_FLAG_26) && ((dynaPoly && dynaPoly->actor.id != 0x108) || dynaPoly == NULL)) {
```

Comment:
This is probably just a single one, so could you do the id here too?

Reply:
Good catch, I fixed it